fuse/all.py

```diff
@@ -290,7 +290,7 @@ def fuse_award():
     app_df.to_csv(deba.data("fuse/appeals.csv"), index=False)
     uof_citizen_df.to_csv(deba.data("fuse/uof_citizens.csv"), index=False)
     uof_officer_df.to_csv(deba.data("fuse/uof_officers.csv"), index=False)
-    award_df.to_csv(deba.data("fuse/award.csv"), index=False)
+    award_df.to_csv(deba.data("fuse/awards.csv"), index=False)
```

Comment:
Perhaps you need to fix the filename in the `.wrgl/config.yaml` file as well?

Reply:
Yes. Thank you!